ArduPlane/quadplane.cpp

```diff
@@ -1824,7 +1824,8 @@ void QuadPlane::vtol_position_controller(void)
 void QuadPlane::setup_target_position(void)
 {
     const Location &loc = plane.next_WP_loc;
-    Location origin = inertial_nav.get_origin();
+    Location origin;
+    ahrs.get_origin(origin);
```

Comment:
These quadplane.cpp changes should go in a commit that is separate from the Copter changes.